### PR TITLE
[16.0][MIG] mail_notification_custom_subject: Migration to 16.0

### PR DESCRIPTION
**Pull request by santostelmo**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Version change should be to 16.0.1.0.0, and it needs to be in a separate commit. Check https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0 for the procedure.

**Review by rfranky — APPROVED** [review]
Tested and Working

File: mail_notification_custom_subject/migrations/16.0.1.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        """
+        UPDATE mail_message_custom_subject
+        SET subject_template=REPLACE(
+                REPLACE(
+                subject_template,
+                '${',
+                '{{'),
+                '}',
+                '}}'
+            )
```
Comment (cuongnmtm) [inline mail_notification_custom_subject/migrations/16.0.1.0.0/post-migration.py:18]: This migration script is used to migrate 14 > 15, I think we could remove this. (remove migrations folder)

> Remove any possible migration script from previous version (in a nutshell, remove migrations folder inside the module if exists). https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0

**Review by cuongnmtm — COMMENTED** [review]
LGTM (code review).

**Pull request closed without merging**